docs/intro.md

```diff
@@ -34,6 +34,14 @@ The admin interface is a single-page app with the entry point stored in a static
 </html>
 ```
 
+The example above uses the latest stable released version of Netlify CMS. If you want to a different release, you can explicitly declare it by editing `@~0.4`. If you want to use the latest version of Netlify CMS you can just omit the version declaration as such:
```

Comment:
Second sentence: "If you want to **use** a different release," - the word "use" is omitted.

Once that's fixed, this is good to merge.

Thanks @guayom!